src/Components/Aspire.MongoDB.Driver/Aspire.MongoDB.Driver.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <PackageTags>$(ComponentDatabasePackageTags) MongoDb</PackageTags>
+    <Description>A generic MongoDb client that integrates with Aspire.</Description>
+    <PackageIconFullPath>$(SharedDir)redis-cube-red_white-rgb.png</PackageIconFullPath>
```

Comment:
Can just delete this line for now. We'll need to investigate permissions required to use the mongodb logo for this package.

Reply:
I've deleted the line.